MaxText/decode.py

```diff
@@ -266,6 +266,7 @@ def main(argv: Sequence[str]) -> None:
   os.environ["TF_FORCE_GPU_ALLOW_GROWTH"] = "true"
 
   validate_config(pyconfig.config)
+  max_utils.write_config_for_gcs(pyconfig.config)
```

Comment:
Sorry this should be in pyconfig right? We don't want to duplicate it everywhere?

Reply:
Done. Changed it to upload in pyconfig.